tests/libvmifact/factory.go

```diff
@@ -50,16 +50,19 @@ func NewFedora(opts ...libvmi.Option) *kvirtv1.VirtualMachineInstance {
 
 // NewCirros instantiates a new CirrOS based VMI configuration
 func NewCirros(opts ...libvmi.Option) *kvirtv1.VirtualMachineInstance {
-	// Supplied with no user data, Cirros image takes 230s to allow login
-	withNonEmptyUserData := libvmi.WithCloudInitNoCloud(libvmici.WithNoCloudEncodedUserData("#!/bin/bash\necho hello\n"))
```

Comment:
nice change! I was just trying to remove some function from test utils, that used another cloudinit data and this code was problematic! thanks for solving it